src/axis.ts

```diff
@@ -177,6 +175,8 @@ export const AXIS_PROPERTY_TYPE: {
   titleX: 'main',
   titleY: 'main',
 
+  tickWidth: 'both',
+  tickCount: 'both',
```

Comment:
1) This index should have alphabetical order, so the output sorts axis properties nicely.  Plus, doing so would remove all the unnecessary example diff. 

2) Can you explain why tickWidth should be in the grid axis?